### PR TITLE
Configure default timezone to Europe/Paris

### PR DESCRIPTION
**Pull request by tauffredou**

### CODE REVIEW

File: jenkins/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+USER root
+RUN export DEBIAN_FRONTEND=noninteractive; \
+    export DEBCONF_NONINTERACTIVE_SEEN=true; \
+    apt-get update -qqy && apt-get install -qqy --no-install-recommends tzdata \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ENV TZ Europe/Paris
+
+USER jenkins
```
Comment (lukkor) [inline jenkins/Dockerfile]:
```suggestion
USER jenkins:jenkins
```

**Comment by lerrigatto** [comment]
I do not understand why we would need to put jenkins in a timezone that is not UTC.
If we look for a build or release timing to correlate to any infrastructure issue or event it will mess up a lot having different timezones.

**Comment by tauffredou** [comment]
It's more relevant to sync date within the application. It's use by people who don't think in UTC and their browser use their locale anyway.
So our Jenkins should use Europe/Paris as default timezone

**Comment by tauffredou** [comment]
@jobteaser/coretech despite @lerrigatto being against, could we state on this PR ?

**Comment by tauffredou** [comment]
@jobteaser/coretech up

**Comment by tauffredou** [comment]
> Is `-Duser.timzone` not sufficient? [jenkinsci/docker#45](https://github.com/jenkinsci/docker/issues/45)

That is java specific. I'd prefer using timezone at system level, but yes it would do the same.

This PR is 1 month old, how can we progress ?

**Pull request closed without merging**